### ISSUE TITLE
Re-implement --update

### ISSUE THREAD

**Issue by gbrindisi** [gbrindisi/xsssniper#1]
Removed after switching from hg to git.

**Comment by fgeek** [comment]
In my opinion in scope of this program it might be better to just use Git for updates.

**Comment by gbrindisi** [comment]
Yes. Also given the small number of updates and the future packaged install this feature will be obsolete.
